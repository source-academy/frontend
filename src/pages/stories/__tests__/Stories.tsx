// import { shallowRender } from 'src/commons/utils/TestUtils';

// import Stories from '../Stories';

// test('Stories renders correctly', () => {
//   const Element: React.FC = () => <Stories />;
//   const tree = shallowRender(<Element />);
//   expect(tree).toMatchSnapshot();
// });
