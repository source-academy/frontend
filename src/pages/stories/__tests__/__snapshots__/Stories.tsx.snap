// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Stories renders correctly 1`] = `<Stories />`;
